spec/factories/registration.rb

```diff
@@ -3,6 +3,7 @@
 FactoryGirl.define do
   factory :registration do
     user
-    conference
+    conference { create(:conference, registration_period: create(:registration_period, start_date: 3.days.ago)) }
```

Comment:
My reasoning behind adding this here was that a registration object can't be created unless the conference is open. It is kinda implicit that if we are creating a registration we meant to do it for an open conference.

Related thought:
Adding a *validation* to `registration` model for `conference` being open is not a bad idea in this situation (where we were able to create invalid registrations for closed conferences). I didn't add the validation cause things work just fine without it, except in tests. In tests we can potentially create registration, which would be invalid. Hence, to avoid adding validation to registration model which we would have never used I added this here. I guess best practice would be to add the validation anyway?
Let me know what do you guys think.